Source/FakeItEasy.Tests/Core/DummyFactoryTests.cs

```diff
@@ -37,6 +38,24 @@ public void Create_should_guard_against_bad_type_argument()
                 .And.ParamName.Should().Be("type");
         }
 
+        [Test]
+        public void Provided_factories_should_have_negative_priority()

```

Comment:
I assume this test is intended for hypothetical factories we would add in the future?